### PR TITLE
A Pull Request for Desktop version branch, a desktop version for the portfolio

### PR DESCRIPTION
**Pull request by KALUNGI-J-BOBSON**
Added the index.html file containing the contents of the web page to be displayed.

Added the style.css file top style how the contents in the index.html file will be displayed according to the desktop view.
Setup the linters for the index.html and style.css to remove errors.
Added the MIT license

### CODE REVIEW

**Review by levy002 — COMMENTED** [review]
## Hello @KALUNGI-J-BOBSON 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request :red_circle:  :red_circle:   :red_circle:  

You have submitted a project, that has conflicts. Kindly solve the conflicts and submit again.

  image

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by KALUNGI-J-BOBSON** [comment]
Hello Dear Levy, I tried to Resolv the issue can y please guide me how i can fix itUrakozeSent from my HeartlessOn Dec 10, 2022, at 20:53, levy_ukwishaka ***@***.***> wrote:﻿
@levy002 commented on this pull request.

Hello @KALUNGI-J-BOBSON
While you made a great effort in this project, unfortunately, I cannot proceed to review your code.
Invalid Code Review Request 🔴  🔴   🔴
You have submitted a project, that has conflicts. Kindly solve the conflicts and submit again.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏
Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.
Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

Invalid Code Review Request does not count into the code reviews limit.

File: index.html [hunk 1]
```diff
@@ -4,317 +4,345 @@
...
+            <h2>TONIC</h2>
+            <ul>
+              <li>CANOPY</li>
+              <li><img src="./image/image3.png" alt="count1" /></li>
+              <li>Back End Dev</li>
+              <li><img src="./image/image4.png" alt="count2" /></li>
+              <li>2015</li>
+            </ul>
+            <p>
+              A daily selecion of privately personalized reads; no accounts or
```
Comment (Alfredbis29) [inline index.html:74]: - [ ] I can see that you put a lot of effort into this project but you still need to improve on something for your project to look so professional, as required for this project your design should match the Figma design, and if we compare your design to the Figma one they not really similar, kindly fix this :point_down: 
 
The `font-weight` of `header` is not the same one we can see on figma design, even the image of the project looks strange even your `p`
your design
derange

 
Figma design
figmacard

File: style.css [hunk 2]
```diff
@@ -1,450 +1,1097 @@
...
+
+    /* or 130% */
+
+    align-items: center;
+
+    /* Neutrals/N0 */
+    color: #fff;
+  }
+
+  .info-form {
```
Comment (Alfredbis29) [inline style.css]: - [ ] Your mobile version is affected and which makes your design not look professional, any time you change something in your design check on your mobile version if it all still working as required, kindly fix this for your project to be more attractive in both mobile version and desktop version ,as required for this project the  mobile design still looks like expected :wink: 

Check this :point_down: 
fffo

File: style.css [hunk 3]
```diff
@@ -1,450 +1,1097 @@
...
+    color: #396df2;
+    box-sizing: border-box;
+    width: 150px;
+    padding: 12px 3px;
+    background: #fff;
+    border: 1px solid #435c9d;
+    border-radius: 14px;
+    margin-top: 3rem;
+    transition: color 0.5s ease;
+  }
```
Comment (Alfredbis29) [inline style.css]: - [ ] I can see that you put some hover on the button but isn't applied kindly fix this for your project to look attractive and recruiters can reach you out for the good work you did :wink: 
your design
butttttt

Figma design 
goot

**Review by Alfredbis29 — CHANGES_REQUESTED** [review]
Hi @KALUNGI-J-BOBSON , 

Good job so far, I can see that you put in a lot of energy into this :+1: 
However, there are some few issues you need to work on in order to proceed to the next project, but believe me, you are almost there. 

 soclose 

## Required Changes :recycle: 
Check under the review

## Optional Suggestions
_Check under the review

Cheers and Happy coding :clap: :clap: :clap: 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @alfredbis29  in your question so I can receive the notification.


#### Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

#
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request closed without merging**